client/data/marketplace/use-marketplace-reviews.ts

```diff
@@ -143,6 +155,16 @@ const deleteReview = ( {
 	} );
 };
 
+const fetchMarketplaceReviewsStats = ( {
+	productType,
+	slug,
+}: ProductProps ): Promise< MarketplaceReviewsStatsResponse | ErrorResponse > => {
+	return wpcom.req.get( {
```

Comment:
I may be mistaken since I'm not completely familiar with React Query, but I don't see any error handling in this code. I'm curious about how the error responses will be managed.

Reply:
`wpcom.req.get` returns a Promise and rejects it if the request returns an error (4XX or 5XX). React Query picks up on the rejected Promise, and sets the `data`, `isError`, and `error` params of the hook it returns based on that.

Reply:
Exactly, in this scenario, we could check if there is any errors in [this line](https://github.com/Automattic/wp-calypso/blob/4eeb72b1c60c526ac6edb969b49212567b426643/client/my-sites/marketplace/components/reviews-summary/index.tsx#L21-L22)